### DISCUSSION TITLE
how to sort array with x-for in template?

### DISCUSSION THREAD

**Discussion by atulmahankal** [alpinejs/alpine#3465] [1. Help]
**code:**
```
<ul x-data="{ fruites: ['Banana', 'Mango', 'Apple'] }">
    <template x-for="fruite in fruites">
        <li x-text="fruite"></li>
    </template>
</ul>
```
**Result:**
<ul>
    <li>Banana</li>
    <li>Mango</li>
    <li>Apple</li>
</ul>

**Required:**
for assending
<ul>
    <li>Apple</li>
    <li>Banana</li>
    <li>Mango</li>
</ul>

for desending
<ul>
    <li>Mango</li>
    <li>Banana</li>
    <li>Apple</li>
</ul>

How can I do that?

Thanks in advance.

**Comment by ekwoka** [comment]
The same way you do in JS.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
